dtreeviz/trees.py

```diff
@@ -1492,23 +1492,27 @@ def _ctreeviz_univar(shadow_tree,
             y_noise = np.random.normal(mu + i * class_step, sigma, size=len(bucket))
             ax.scatter(bucket, y_noise, alpha=colors['scatter_marker_alpha'], marker='o', s=dot_w, c=color_map[i],
                        edgecolors=colors['scatter_edge'], lw=.3)
+    else:
+        raise ValueError(f'Unrecognized gtype = {gtype}!')
 
     ax.tick_params(axis='both', which='major', width=.3, labelcolor=colors['tick_label'],
                    labelsize=fontsize)
 
     splits = [node.split() for node in shadow_tree.internal]
     splits = sorted(splits)
-    bins = [ax.get_xlim()[0]] + splits + [ax.get_xlim()[1]]
 
-    if 'splits' in show:  # this gets the horiz bars showing prediction region
+    if 'preds' in show:  # this gets the horiz bars showing prediction region
```

Comment:
So this looks like it's renaming rather than adding functionality?

Reply:
OK I get it now. You are separating the dashed vertical lines from the prediction horizontal boxes.

Reply:
Yeah for 2 reasons.

1. As you note below, the prediction horizontal boxes overlap the bottom of the bars.
2. It would be nice to turn the prediction boxes off when plotting the splits of a larger (more than 1-2 feature) tree, as the prediction regions could overlap.

Reply:
predictions in a uni-var feature space shouldn't have any overlap, should there?  Every rectangular region inscribed by a decision tree's leaf node(s) will have a unique prediction I think.